pkg/dronereceiver/handler.go

```diff
@@ -67,24 +37,22 @@ func (d *droneWebhookHandler) handler(resp http.ResponseWriter, req *http.Reques
 	// However this appears to be sent when a build completes; The structure however changes slightly as in
 	// the `repo.build` seems to be absent in the first one.
 	// TODO: Revisit this, we may not need the Finished check.
-	if evt.Repo.Build == nil || evt.Repo.Build.Finished == 0 {
-		return fmt.Errorf("no build info provided from the webhook event: %v", err)
+	if build == nil || build.Finished == 0 {
+		logger.Error("no build info provided from the webhook event")
```

Comment:
should we error here? I guess we could Warn?

Reply:
done in 96d49d6